spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -292,6 +294,24 @@
           expect(proofing_component.threatmetrix).to equal(true)
           expect(proofing_component.threatmetrix_review_status).to eq('pass')
         end
+
+        context 'nil response body from ddp' do
+          let(:ddp_result) { Proofing::Result.new }
```

Comment:
I think it would be nice to explain in a comment that the response_body tacitly becomes nil here

Reply:
updated in a9f4b4883c6401d9820de435f10e96cdf34aa6ca